src/physlr-indexlr.cc

```diff
@@ -59,16 +77,18 @@ static void
 printUsage(const std::string& progname)
 {
 	std::cout << "Usage:  " << progname
-	          << "  -k K -w W [-v] [-o FILE] FILE...\n\n"
-	             "  -k K       use K as k-mer size\n"
-	             "  -w W       use W as sliding-window size\n"
-	             "  --pos      include minimizer positions in the output\n"
-	             "  --strand   include minimizer strand in the output\n"
-	             "  -v         enable verbose output\n"
-	             "  -o FILE    write output to FILE, default is stdout\n"
-	             "  -t N       use N number of threads (default 1, max 5)\n"
-	             "  --help     display this help and exit\n"
-	             "  FILE       space separated list of FASTQ files\n";
+	          << "  -k K -w W [-r repeat_bf_path] [-s solid_bf_path] [-v] [-o FILE] FILE...\n\n"
+	             "  -k K        use K as k-mer size\n"
+	             "  -w W        use W as sliding-window size\n"
+	             "  -r repeat_bf_path  use a bloomfilter to filter out repetitive minimizers\n"
+	             "  -s solid_bf_path  use a bloomfilter to filter in solid minimizers\n"
```

Comment:
```suggestion
	             "  -s solid_bf_path  use a Bloom filter only select solid minimizers\n"
```
Just thinking 'filter in' doesn't sound right